README.md

```diff
@@ -37,16 +37,67 @@ int main( )
     vtu11::DataSet pointDataSet { "Temperature", 1, pointData };
     vtu11::DataSet cellDataSet { "Conductivity", 1, cellData };
 
-    // Create small proxy mesh type 
-    vtu11::Vtu11UnstructuredMesh mesh { points, connectivity, offsets, types };
+    // Create small proxy mesh type
+    vtu11::Vtu11UnstructuredMesh mesh
+    {
+        points,       // coordinates of the points
+        connectivity, // connectivities of the cells
+        offsets,      // offsets between the cell points
+        types         // types of the cells, see "https://vtk.org/wp-content/uploads/2015/04/file-formats.pdf"
+    };
 
     // Write data to .vtu file using raw binary appended format
-    vtu11::write( "test.vtu", mesh, { pointDataSet }, { cellDataSet }, 
-                  vtu11::RawBinaryAppendedWriter { } );
+    vtu11::write(
```

Comment:
I don't like this format but if you really want we can keep it.

Reply:
initially I wanted to add explanations to the arguments (like I did in the parallel example) but then I didn't
I can revert, but then it is inconsistent with the parallel example :D